terraform/main.tf

```diff
@@ -0,0 +1,32 @@
+provider "aws" {
+  version             = "~> 1.27.0"
+  region              = "us-east-1"
+  profile             = "hca"
+}
+# cant interpolate the key for the s3 bucket -> need to hard code for each deployment stage/environment
+terraform {
+  required_version = "~>0.11.7"
+
+  backend "s3" {
+    bucket = "org-humancellatlas-861229788715-terraform"
+    key = "query-service/dev/state.tfstate"
```

Comment:
I would parameterize these so that, eventually, we can deploy this to different environments and accounts. Use the `-backend-config` option in `terraform init`.

Reply:
👍